src/components/layout/navigation.tsx

```
@@ -0,0 +1,23 @@
+import { Link } from '@tanstack/react-router'
```

ESLint

[error] Unable to resolve path to module '@tanstack/react-router'.

(import-x/no-unresolved)